edge/storage.py

```diff
@@ -88,5 +88,9 @@ def setup_storage(project_id: str, region: str, bucket_name: str) -> StorageBuck
                     bucket_name,
                 )
             return StorageBucketState(bucket_path)
+        except NotFound as e:
+            raise EdgeException(f"The '{project_id}' project could not be found. It might mean that the quota project "
+                                f"is set to a different project. Please generate new application default "
```

Comment:
different project to what? the one set by `gcloud config set project`.

I'd add that context. I also think we could have some content in the repo for 'known issues' or the troubleshooting guide we discussed before. Then such error messages can reference this. But that's not something for this pull request.